sklearn/model_selection/tests/test_split.py

```diff
@@ -410,11 +410,11 @@ def test_stratified_kfold_sparsetarget():
         for tup_d, tup_s in zip(splits_dense, splits_sparse):
             assert_array_equal(tup_d[0], tup_s[0])
             assert_array_equal(tup_d[1], tup_s[1])
-    # Ensure that StratifiedKFold raises an error when the sparse y does not
-    # have a shape of the form (n_samples,) or (n_samples,1).
-    y_sparse_large = csr_matrix((n_samples, 1000))
+    # Ensure that StratifiedShuffleSplit raises an error when y has shape
+    # (n_samples, n_labels) with n_labels > 1.
+    y_sparse_multilabels = csr_matrix((n_samples, 3))
     assert_raises(ValueError, next,
-                  StratifiedKFold(n_splits=2).split(_X, y_sparse_large))
+                  StratifiedKFold().split(_X, y_sparse_multilabels))
```

Comment:
Comment refers to `StratifiedShuffleSplit` but code uses `StratifiedKFold`.

(Test is called `test_stratified_kfold_sparsetarget`)

Reply:
@thomasjpfan Thank you for spotting this !